dashboard/src/main/kotlin/com/ritense/dashboard/autoconfigure/DashboardAutoConfiguration.kt

```diff
@@ -71,13 +75,17 @@ class DashboardAutoConfiguration {
         widgetConfigurationRepository: WidgetConfigurationRepository,
         userManagementService: UserManagementService,
         widgetDataSourceResolver: WidgetDataSourceResolver,
+        authorizationService: AuthorizationService,
+        @Value("\${valtimo.authorization.dashboard.enabled:false}") authorizationEnabled: Boolean
```

Comment:
Since we are configuring the dashboard, I would expect the configuration of it to be at:
`valtimo.dashboard.authorization.enabled` instead.

It's a bit of a grey area, because you could argue that we are configuring authorization here. @marijnritense ?

Reply:
I think it's fine either way. My suggestion would be to just keep this, and just mark the code that's of temporary nature here with a TODO so we know to remove it when major 13 comes along.